hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/client/HdfsClientConfigKeys.java

```diff
@@ -79,6 +79,8 @@
   String  DFS_NAMENODE_HTTPS_ADDRESS_KEY = "dfs.namenode.https-address";
   String DFS_HA_NAMENODES_KEY_PREFIX = "dfs.ha.namenodes";
   int DFS_NAMENODE_RPC_PORT_DEFAULT = 8020;
+  String DFS_OBSERVER_READ_ENABLE = "dfs.observer.read.enable";
+  boolean DFS_OBSERVER_READ_ENABLE_DEFAULT = true;
```

Comment:
Hi @simbadzina , should the default here be false?

Reply:
A default of false preserves client behavior of sending the actual state ID. When this is true the client sends "-1" as its last seen state ID.

Reply:
Thank you for your explanation. I understand.